LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2014-2024, Matthew Brett and the Delocate contributors.
+Copyright (c) 2014-2025, Matthew Brett and the Delocate contributors.
```

Comment:
Time to add your name here I think.

Reply:
As a Delocate contributor I'm under the impression that I'm already a copyright holder along with all other contributors and those in the AUTHORS file (which I'm already listed in).

But I understand that my contributions would be considered significant by this point, so I'll at least add myself as a maintainer in the pyproject.toml file.